### DISCUSSION TITLE
매주 저축금액 만원단위로 반올림 할 것인지?

### DISCUSSION THREAD

**Discussion by limhyoseok** [DeveloperAcademy-POSTECH/toGather#67] [General]
<img width="338" alt="image" src="https://user-images.githubusercontent.com/59302419/173955733-77b4d3e0-9b6c-41fa-b918-5918e3c56ba3.png">

에어팟의 경우 30만원으로 매주 저축금액이 6천원(= 30만/48)이다. 
만원 단위로 자를경우 반올림 처리를 해야하고 결과적으로 매달 저축금액 1만원이되면 1만원 * 48 =50만원으로 애플워치 금액과 비슷해진다. 

소수점을 넣어서 천원자리까지 보여줄 경우 디자인적으로 괜찮을지?
<img width="612" alt="image" src="https://user-images.githubusercontent.com/59302419/173955866-9c08218e-9a02-4945-9046-bd49529e7e99.png">

**Comment by yeongwooCho** [comment]
저는 올림이 좋다고 생각합니다!
UI가 간결해 보이고 송금하는 행위를 생각 해볼때 천원 단위까지 입력하는 것을 귀찮음이 생길 것 같습니다!

**Comment by limhyoseok** [comment]
2022.06.16 결론
천원단위까지 보여주는 걸로 결정
